src/audits/FocusableElementNotVisibleAndNotAriaHidden.js

```diff
@@ -43,7 +43,7 @@ axs.AuditRules.addRule({
         // Ignore elements which have a negative tabindex and no text content,
         // as they will be skipped by assistive technology
         var textAlternatives = axs.properties.findTextAlternatives(element, {});
-        if (!textAlternatives || textAlternatives.trim() === '')
+        if (textAlternatives == null || textAlternatives.trim() === '')
```

Comment:
Let's use === null - closure compile should catch if `findTextAlternatives` weirdly returns undefined anywhere.